### ISSUE TITLE
[time.syn] Unclear namespace prefix for literal operators

### ISSUE THREAD

**Issue by jensmaurer** [cplusplus/draft#3328]
Why do the literal operators (such as operator ""s) shown in the header synopsis have a qualified-id as the return type?

Ordinary unqualified lookup would/should find std::chrono::month from a subnamespace of std::chrono, too, I thought.

This relates to consistency issues in the itemdecls of operator""y and operator""d, raised by 
JP338 cplusplus/nbballot#334 and JP 343 cplusplus/nbballot#339.

**Comment by CaseyCarter** [comment]
The `<chrono>` synopsis is in [time.syn] - there is no subclause [chrono.syn]. I also have no explanation for why many of the names are qualified - e.g. `struct common_type<chrono::duration` in namespace `std::chrono` need not be qualified - other than that `std::chrono` was one of our first nested namespaces and LWG policy for how to specify components in sub-namespaces of `std` was less developed than today.

**Comment by jensmaurer** [comment]
std::filesystem is way older than std::chrono, yet we still don't describe the "leave off namespaces" presentation form properly.

**Comment by CaseyCarter** [comment]
> std::filesystem is way older than std::chrono, yet we still don't describe the "leave off namespaces" presentation form properly.

`std::chrono` was added in C++11, `std::filesystem` in C++17. The example I pointed out above is completely invalid, however, since it's a specialization of `common_type` in namespace `std` and therefore *must* qualify `chrono::duration`.

Let's start over: The user-defined literals in `<chrono>` are defined in namespace `std::literals::chrono_literals` which is not a subnamespace of `std::chrono`, so the return types must be `chrono::`-qualified; they would not be found by unqualified lookup.
